README.md

```diff
@@ -52,3 +52,83 @@ Se você perceber algo de errado ou faltando, por favor abra uma [issue](https:/
   - ⚠️ Esta é uma nova cheatsheet, toda a assistência é bem-vinda.
 
 ---
+
+### Tabela de conteúdo da Cheatsheet básica
```

Comment:
Awesome!! Just a small detail here: could you update "conteúdo" to "conteúdos"?

Reply:
Sure. It is already corrected it in my next commit.